src/NUnitTestAdapter/NUnit3TestAdapter.csproj

```diff
@@ -54,9 +54,11 @@
       <HintPath>$(MSBuildProgramFiles32)\Microsoft Visual Studio 11.0\Common7\IDE\CommonExtensions\Microsoft\TestWindow\Microsoft.VisualStudio.TestPlatform.ObjectModel.dll</HintPath>
       <Private>False</Private>
     </Reference>
-    <Reference Include="nunit.core.engine, Version=3.0.5610.33198, Culture=neutral, PublicKeyToken=2638cd05610744eb, processorArchitecture=MSIL">
-      <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\..\packages\NUnit.Core.Engine.3.0.0-beta-2\lib\nunit.core.engine.dll</HintPath>
+    <Reference Include="nunit.engine">
```

Comment:
Yes, I think we should. Right now though we don't have an engine package.
On Nov 8, 2015 2:58 PM, "Rob Prouse" <notifications@github.com> wrote:

> In src/NUnitTestAdapter/NUnit3TestAdapter.csproj
> <https://github.com/nunit/nunit3-vs-adapter/pull/61#discussion_r44232295>:
>
> > @@ -54,9 +54,11 @@
> >        <HintPath>$(MSBuildProgramFiles32)\Microsoft Visual Studio 11.0\Common7\IDE\CommonExtensions\Microsoft\TestWindow\Microsoft.VisualStudio.TestPlatform.ObjectModel.dll</HintPath>
> >        <Private>False</Private>
> >      </Reference>
> > -    <Reference Include="nunit.core.engine, Version=3.0.5610.33198, Culture=neutral, PublicKeyToken=2638cd05610744eb, processorArchitecture=MSIL">
> > -      <SpecificVersion>False</SpecificVersion>
> > -      <HintPath>..\..\packages\NUnit.Core.Engine.3.0.0-beta-2\lib\nunit.core.engine.dll</HintPath>
> > +    <Reference Include="nunit.engine">
>
> Fine for now I guess since this is holding up other fixes, but are we
> going to switch it back to NuGet before release?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nunit/nunit3-vs-adapter/pull/61/files#r44232295>.
>